pkg/pod/network.go

```diff
@@ -58,3 +65,18 @@ func StaticIPAnnotationWithInterfaceMacAndNamespace(
 
 	return baseAnnotation
 }
+
+// StaticIPBondAnnotationWithInterface defines static name for bonded interfaces and name, interface and IP for the
+// main bond int.
+func StaticIPBondAnnotationWithInterface(
+	bondNadName, bondIntName string, sriovNetworkNameList, ipAddrBond []string) []*multus.NetworkSelectionElement {
```

Comment:
This function has to return just bond annotation and not list of annotations:
*multus.NetworkSelectionElement